Sources/SwiftParser/Attributes.swift

```diff
@@ -816,7 +816,8 @@ extension Parser {
         elements.append(
           .labeledSpecializeEntry(
             RawLabeledSpecializeEntrySyntax(
-              label: ident,
+              RawUnexpectedNodesSyntax([ident], arena: self.arena),
+              label: RawTokenSyntax(missing: .identifier, arena: self.arena),
```

Comment:
This is not correct. 
But I didn't know how we can add a missing token that is one of the correct choices.